### DISCUSSION TITLE
Cleanup of issues

### DISCUSSION THREAD

**Discussion by nighcoder** [clojupyter/clojupyter#156] [General]
As you may have noticed, I've started moving some Issues to Discussions better keep track of opened issues while keeping the thread open for feedback and comments.

While trying to move the announcements for versions 0.3.0 -> 0.3.2, I've got the following error:

> Unable to convert this issue to a discussion. Category is not accessible to the actor

I've tried to move it to **Announcements** category, yet I'm not sure if the actor in this error refers to me or its author, or if the problem is me attempting to move someone else's thread.

@krukow, are there some limits on who can open threads on **Announcements**? I would assume there must be some, so new users can't open threads there by default, but I can't see any settings. Do you have access to fine tune the settings?

**Comment by antn** [comment]
Hey @nighcoder! I'm an engineer from the Discussions team at GitHub. Sorry that you ran into some issues when trying to convert issues to discussions in the Announcements category!

I've just shipped a fix for this, so can you try it again and let me know if you're still having any problems?

**Comment by nighcoder** [comment]
@antn, I've just tried it and it works. Thanks for fixing it!
